src/main/scala/org/ergoplatform/nodeView/ErgoModifiersCache.scala

```diff
@@ -30,7 +30,7 @@ class ErgoModifiersCache(override val maxSize: Int)
       history
         .headerIdsAtHeight(history.fullBlockHeight + 1)
         .flatMap(id => history.typedModifierById[Header](id))
-        .flatMap(_.sectionIds.map(id => mutable.WrappedArray.make[Byte](id)))
+        .flatMap(_.sectionIds.map(id => mutable.WrappedArray.make[Byte](id._2)))
```

Comment:
it's hard to read this line. I should somehow remember what that `sectionIds` are tuples of type `ModifierTypeId` -> `ModifierId`. Could we introduce in future (not in this pull request) a special case class for these tuples as they occur often in code? For now we can at least leave a comment on what doest `id._2` mean

Reply:
I don't like to write comments here, you can click to `sectionIds` and see the type. Case class might be useful, but will also consume more resources I guess